definitions/scenarios/satellite_upgrade.rb

```diff
@@ -63,12 +94,13 @@ def compose
         modules_to_enable = ["#{feature(:instance).downstream.module_name}:#{el_short_name}"]
         add_step(Procedures::Packages::EnableModules.new(:module_names => modules_to_enable))
       end
-      add_step(Procedures::Packages::Update.new(
-        :assumeyes => true,
-        :download_only => true
-      ))
-      add_step(Procedures::Service::Stop.new)
-      add_step(Procedures::Packages::Update.new(:assumeyes => true, :clean_cache => false))
+      add_steps(
+        Procedures::Packages::Update.new(
+          :assumeyes => true,
+        ),
```

Comment:
I think this was lost in a rebase:
```suggestion
        Procedures::Packages::Update.new(
          :assumeyes => true,
          :download_only => true,
        ),
```